.pipelines/trigger.yaml

```diff
@@ -0,0 +1,51 @@
+pr:
+  branches:
+    include:
+      - master
+      - release/*
+  paths:
+    exclude:
+      - ".devcontainer"
+      - ".hooks"
+      - ".vscode"
+      - ".github"
+      - docs
+
+
+trigger:
+  branches:
+    include:
+      - gh-readonly-queue/master/*
```

Comment:
Until we have hooks back into the PR we might want to turn off this trigger as all it will be doing is raising COGS.

Reply:
We want this to run so we can validate parity. We can disable the pipeline if COGS becomes a problem.